api/core/tools/utils/web_reader_tool.py

```diff
@@ -356,3 +356,16 @@ def content_digest(element):
                 digest.update(child.encode("utf-8"))
             digest = digest.hexdigest()
     return digest
+
+
+def get_image_upload_file_ids(content):
+    pattern = r"!\[image\]\((.*?file-preview)\)"
```

Comment:
You need to also find the pattern contains `image-preview`

Reply:
Okay, I have modified code and made a commit, thanks.